docs/playground/agent-tbox.tsx

```
@@ -0,0 +1,581 @@
+        role: 'assistant',
+      };
+    },
+    transformMessage: (info) => {
+      const { originMessage, chunk } = info || {};
+      let parsedPayload
+      try {
+        parsedPayload = JSON.parse((chunk as any).data.payload);
+      } catch (e) {
+        //
+      } 
+      let content = originMessage?.content || '';
```

Biome (2.1.2)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)